Gemfile

```diff
@@ -87,8 +87,6 @@ gem 'jquery-ui-rails', '~> 4.2.1'
 # for languages validation
 gem 'iso-639'
 
-gem 'thor', '~> 0.19'
```

Comment:
We don't need _thor_ anymore?

Reply:
no, as I say in the commit message, thor was set to 1.19.1 because of a bug, but it is not longer needed. Check #1664